.travis.yml

```diff
@@ -65,6 +65,8 @@ script:
     echo "Can't run cloud tests without keys so don't run tests";
   else
     bash scripts/install_git_lfs.sh;
+    ./gradlew installDist;
```

Comment:
This was a change to help debug what was going on by running `PrintReads` and including it's output in the travis log.  I left it because I figured it was a useful thing to have and it adds very little to the test run time.